node-typescript/sync-with-meilisearch/src/utils.ts

```diff
@@ -0,0 +1,27 @@
+import path from "path";
+import { fileURLToPath } from "url";
+import fs from "fs";
+
+export function throwIfMissing(obj: any, keys: string[]): void {
```

Comment:
Can we add docstring descriptions to all the functions here:
e.g. for throwIfMissing
```typescript 
/**
* Throws an error if any of the keys are missing from the object
*/
```

Reply:
@loks0n Do i write the descriptions for all the main.ts functions too ?

Reply:
Just the `utils.ts` file. 

We want to keep the template consistent between runtimes, and this is how the node javascript template looks :)

You can use the same descriptions as the node template also